src/Api/Controllers/UsersAssociatedAppsController.php

```diff
@@ -44,4 +46,28 @@ public function onConstruct()
             ['companies_id', ':', $this->userData->currentCompanyId()]
         ];
     }
+
+    /**
+     * Change user's active status
+     *
+     * @param int $usersId
+     *
+     * @return Response
+     */
+    public function changeUserActiveStatus(int $usersId): Response
+    {
+        $userAssociatedApp = $this->model->findFirstOrFail([
```

Comment:
@rwhite27 only admins can do this

Reply:
@rwhite27 lets move te isAdmin to /src/Traits/PermissionsTrait.php to maintain consistency